src/components/Alerts/Alerts.README.md

```diff
@@ -0,0 +1,80 @@
+# Alerts API
+
+Alert can be accessible/render by Alert API, you can import alertAPI from `Alerts.tsx`.
+The API have only one function `open`.
+
+## Usage
+
+```javascript
+import alertAPI as AlertAPI from "./index"
+
+alertAPI.open({
```

Comment:
In my opinion `add` would be better instead of `open` because, `open` implies that something is already present and you are openning it, however in this case, we are creating a new instance of an alert.